__tests__/pattern/pattern.query.test.js

```diff
@@ -0,0 +1,349 @@
+const { gql } = require('apollo-boost');
+/* eslint-disable no-undef */
+const client = require('../../utils/apollo-test-client');
+require('dotenv').config();
+const {
+  patternDoesNotExistId,
+  skip,
+  limit,
+  wrongLimit,
+  wrongSkip,
+  languageTypeName,
+  imageTypeName,
+} = require('./pattern.variables');
+const { PATTERN_NOT_FOUND } = require('../../error-messages/pattern.messages');
+
+let patternId;
+
+const testValue = 't e s t new';
+const patternToAdd = {
+  name: [
+    {
+      lang: 'uk',
+      value: testValue,
+    },
+    {
+      lang: 'en',
+      value: testValue,
+    },
+  ],
+  description: [
+    {
+      lang: 'uk',
+      value: testValue,
+    },
+    {
+      lang: 'en',
+      value: testValue,
+    },
+  ],
+  images: {
+    large: 'large_335nr4j5dkebkw5cy_test.jpg',
+    medium: 'medium_335nr4j5dkebkw5cy_test.jpg',
+    small: 'small_335nr4j5dkebkw5cy_test.jpg',
+    thumbnail: 'thumbnail_335nr4j5dkebkw5cy_test.jpg',
+  },
+  material: 'test',
+  handmade: false,
+  available: true,
+};
+
+beforeAll(async () => {
```

Comment:
This statement should be inside the `describe` statement

Reply:
fixed